src/sinks/elasticsearch/config.rs

```diff
@@ -69,6 +69,12 @@ pub struct ElasticsearchConfig {
     #[serde(default)]
     pub suppress_type_name: bool,
 
+    /// Whether or not to retry successful requests with partial failures.
```

Comment:
Semi-nit but the usage of "with" in the sentence could imply partial failures are being used to retry.

```suggestion
    /// Whether or not to retry successful requests containing partial failures.
```